client/src/templates/Challenges/components/fill-in-the-blanks.tsx

```diff
@@ -53,76 +109,61 @@ function FillInTheBlanks({
     return cls;
   };
 
-  const getAnswerLength = (answer: string): number => {
-    const parsedAnswer = parseAnswer(answer);
-
-    if (typeof parsedAnswer === 'string') {
-      return parsedAnswer.length;
-    }
-
-    // TODO: This is a simplification. Revisit later to account for tones and spaces.
-    return parsedAnswer.pinyin.length;
-  };
-
   const paragraphs = parseBlanks(sentence);
   const blankAnswers = blanks.map(b => b.answer);
 
+  const ariaInputDescription =
+    inputType === 'pinyin-to-hanzi' ? t('aria.pinyin-to-hanzi-input-desc') : '';
+
   return (
     <>
       <ChallengeHeading heading={t('learn.fill-in-the-blank.heading')} />
       <Spacer size='xs' />
+      <p className='sr-only'>{t(ariaInputDescription)}</p>
```

Comment:
Add a guide here to help screen reader users identify which input type a challenge uses. All the inputs are labeled "blank", so the users can’t tell the difference.